File: 2089-find-target-indices-after-sorting-array/2089-find-target-indices-after-sorting-array.py
class Solution:
    def targetIndices(self, nums: List[int], target: int) -> List[int]:
        nums.sort()
        res = []
        for i in range(0, len(nums)):
            if nums[i] == target:
                res.append(i)
            elif nums[i] > target:
                break
        return res

File: 2089-find-target-indices-after-sorting-array/2089-find-target-indices-after-sorting-array.kt
class Solution {
    fun targetIndices(nums: IntArray, target: Int): List<Int> {
        nums.sort()
        val res = mutableListOf<Int>()
        for (i in 0..(nums.size-1)) {
            if (nums[i] === target) {
                res.add(i)
            } else if (nums[i] > target) {
                break
            }
        }
        return res
    }
}